03-other-topics/004-kibana.md

```diff
@@ -4,16 +4,16 @@ This document is intended to assist engineers in accessing application and syste
 
 Production and non-production data will be indexed in seperate clusters. 
 ## Quick start
-Live
-https://kibana.apps.cloud-platform-live-0.k8s.integration.dsd.io/_plugin/kibana
+Live-0 cluster: 
+[https://kibana.apps.cloud-platform-live-0.k8s.integration.dsd.io/_plugin/kibana](https://kibana.apps.cloud-platform-live-0.k8s.integration.dsd.io/_plugin/kibana)
 
-Test
-https://kibana.apps.cloud-platform-test-1.k8s.integration.dsd.io/_plugin/kibana
+Test-1 cluster:
+[https://kibana.apps.cloud-platform-test-1.k8s.integration.dsd.io/_plugin/kibana](https://kibana.apps.cloud-platform-test-1.k8s.integration.dsd.io/_plugin/kibana)
 ## How to get my data into Elasticsearch
 As long as your application is writing to stdout, it'll be collected by fluentd and passed to the Elasticsearch cluster. Use one of the links above and authenticate with your GitHub username and password. 
 ## How do I use Kibana?
 
-https://www.elastic.co/guide/en/kibana/6.3/search.html
+[https://www.elastic.co/guide/en/kibana/6.3/search.html](https://www.elastic.co/guide/en/kibana/6.3/search.html)
```

Comment:
cluster is 6.2 right now, and there are breaking changes in 6.3 so maybe pin to the exact release: https://www.elastic.co/guide/en/elasticsearch/reference/current/release-notes-6.2.3.html